src/Datadog.Trace/Agent/TransportStrategy.cs

```diff
@@ -20,10 +20,10 @@ public static IApiRequestFactory Get(TracerSettings settings)
             switch (strategy)
             {
                 case DatadogTcp:
-                    Log.Information("Using {0} for trace transport.", nameof(TcpStreamFactory));
+                    Log.Debug("Using {FactoryType} for trace transport.", nameof(TcpStreamFactory));
                     return new HttpStreamRequestFactory(new TcpStreamFactory(settings.AgentUri.Host, settings.AgentUri.Port), new DatadogHttpClient());
                 case DatadogNamedPipes:
-                    Log.Information("Using {0} for trace transport, with pipe name {1} and timeout {2}ms.", nameof(NamedPipeClientStreamFactory), settings.TracesPipeName, settings.TracesPipeTimeoutMs);
+                    Log.Debug("Using {FactoryType} for trace transport, with pipe name {PipeName} and timeout {Timeout}ms.", nameof(NamedPipeClientStreamFactory), settings.TracesPipeName, settings.TracesPipeTimeoutMs);
```

Comment:
This type of information is pretty important to see at first glance in environments like AAS.
I have the same worry as not writing the configuraiton information.

Reply:
Reverted this change back to Information